packages/mirrorful/editor/src/components/ColorPalette/ColorPaletteSection.tsx

```diff
@@ -19,6 +20,16 @@ export function ColorPaletteSection({
   onUpdateColors: (newColors: TColorData[]) => void
 }) {
   const { isOpen, onOpen, onClose } = useDisclosure()
+  
+  function handleAddColorCss () {
+    const cssObj = {
+      padding: '18px 0',
+      marginTop: '0'
+    }
+    if(colors.length > 0)
+      cssObj.marginTop = 'var(--chakra-space-32)'
+    return cssObj
+  }
```

Comment:
We can simplify this as such, I think:

```javascript
<Box css={{
   padding: '18px 0',
   marginTop: colors.length > 0 ?  '32px' : '0'
}}/>
```

Reply:
Thanks, please keep the feedback coming - it helps me lift my game!